src/VirtualClient/VirtualClient.Monitors/Nvidia-Smi/NvidiaSmiMonitor.cs

```diff
@@ -39,34 +39,31 @@ protected override Task ExecuteAsync(EventContext telemetryContext, Cancellation
             {
                 try
                 {
-                    if (this.Platform == PlatformID.Unix)
+                    // Check that nvidia-smi is installed. If not, we exit the monitor.
+                    bool toolsetInstalled = await this.VerifyToolsetInstalledAsync(telemetryContext, cancellationToken);
```

Comment:
Do they use exactly the same commandline and output? also no need append .exe on windows? in some scripting environment you need explicitly .exe appended.